playbooks/tasks/portal-docker-services-check-running.yml

```diff
@@ -0,0 +1,123 @@
+---
+# Check docker services are running (not restarting)
+# Inputs:
+# - docker_containers_check_list: List of docker container names to check
+# - timeout_secs: Total timeout in seconds to wait for containers running
+#     successfully, not restarting (expected to be 120+ seconds)
+
+- name: Assert that inputs are defined correctly
+  ansible.builtin.assert:
+    that:
+      - docker_containers_check_list is defined
+      - docker_containers_check_list | length > 0
+      - docker_containers_check_timeout_secs is defined
+      - docker_containers_check_timeout_secs > 0
```

Comment:
Should this be `> min_required_non_restarting_time_in_secs` ? So like, `70` or something? If someone sets it below hardcoded `65` then this playbook will inevitably fail.

Reply:
Updated the condition.